az-digital/az_quickstart
# Closes #1062 Add summary link field to Person content type

## Description
We have functionality on news and events to allow a summary link field to point to an external URL. This PR adds similar functionality to the Person content type.

## Related Issue
#1062

## How Has This Been Tested?
Locally and in Probo
To test:

- Edit a Person
- In the summary tab, add an external URL to the external bio field
- Go to the Person row and grid view pages (/person /person-list
- Verify that clicking on the person you edited now goes to the external URL (both name and photo)
- Go to the actual node (navigate through content)
- Verify that the canonical URL is set to the external link

## Types of changes


## Checklist:
